locales/translations-sorter.js

```diff
@@ -0,0 +1,25 @@
+/**
+ *  Translations sorter
+ *
+ * - author: fatb38
+ * - license: MIT
+ * - Github profile: https://github.com/fatb38
+ */
+
+const fs = require('fs');
+const locales = ['de', 'en', 'es', 'fr', 'it', 'lol', 'nl'];
```

Comment:
```suggestion
const locales = ['de', 'en', 'es', 'fr', 'it', 'nl'];
```
This file will be removed soon